### ISSUE TITLE
getting systemd / services to work in docker container

### ISSUE THREAD

**Issue by averywald** [pwn-ble/pwnable#14]
when trying to run regular commands in the current docker container build, like `sudo systemctl start pwnable.service`, the container fails to do so, spitting out `Failed to get D-Bus connection: Unknown error -1`. 

This is because `bash` is the root process, since we are running the container in interactive mode, ie. `docker run -it pwnable`.

we need to figure out how to initialize systemd.

it has been a known issue that docker and systemd don't play nice together.  a main reason is that containers are meant to run a single service or process and then terminated, and as such, are supposed to be used in parallel with management via Kubernetes or Docker compose, etc.  

bottom line, we need to get the pwnable systemd service working somehow to manage challenge delivery and data.  i would appreciate you guys looking at this [latest commit](https://github.com/pwn-ble/pwnable/commit/0f251ad65804aa27ac42fdaeecfb9427afdc274e), and doing some research and tinkering to try to get this working one way or another.
